Initium-Odp/war/odp/javascript/messager-impl.js

```diff
@@ -57,6 +57,12 @@ messager.onNotificationMessage = function(message)
 
 messager.onChatMessage = function(chatMessage)
 {
+	if (chatMessage.message.toLowerCase().startsWith("/")) {
```

Comment:
To lower case unnecessary here. Also......... Are you sure Js has that function for strings?